test_scripts/WebEngine/028_OnExitApplication_unregister.lua

```diff
@@ -0,0 +1,49 @@
+--[[ General test configuration ]]
```

Comment:
@mvorobio Please add script description.
Example:
```
---------------------------------------------------------------------------------------------------
-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0240-sdl-js-pwa.md
--
-- Description:
-- Processing of the UpdateAppList request to HMI if Web application is unregistered
--
-- Precondition:
-- 1. SDL and HMI are started
-- 2. Web WebApp_1 is enabled through SetAppProperties
-- 3. Web WebApp_2 is enabled through SetAppProperties
-- 4. Web WebApp_1 is registered and activated

-- Sequence:
-- 1. Web app is unregistered
--  a. SDL sends BC.UpdateAppList with WebApp_1 and WebApp_2 since they are still enabled
---------------------------------------------------------------------------------------------------

Reply:
@aderiabin Please check https://github.com/mvorobio/sdl_atf_test_scripts/pull/1/commits/285c8094c242f73fd09d3a3d348e61f456a7e233